.github/workflows/update-datasets-docs.yml

```
@@ -0,0 +1,52 @@
+name: Access Token Request
+
+on:
+  schedule:
+    # Run every Friday at 10 am PST (6 pm UTC)
+    - cron: '0 18 * * 5'
+  workflow_dispatch:
+
+jobs:
+  request-token:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        # Add commands to install any other dependencies your script needs, e.g.:
+        # pip install requests
+
+    - name: Run script
+      env:
+        CLIENT_ID: ${{ secrets.CLIENT_ID }}
+        CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+      run: python ../../rest_api/scripts/datasets.py
```

```suggestion
      run: python ./rest_api/scripts/datasets.py
```

FYI: the CWD of these workflows/steps is going to be the project root unless otherwise specific in the config.